packages/mask/background/services/wallet/services/wallet/index.ts

```diff
@@ -68,6 +77,18 @@ export async function getWalletPrimary() {
     )
 }
 
+export async function getMnemonicId(storedKeyInfo: api.IStoredKeyInfo) {
+    const masterPassword = await password.INTERNAL_getMasterPasswordRequired()
+    const firstWallet = await Mask.createAccountOfCoinAtPath({
+        coin: api.Coin.Ethereum,
+        password: masterPassword,
+        derivationPath: `${HD_PATH_WITHOUT_INDEX_ETHEREUM}/0`,
+        StoredKeyData: storedKeyInfo.data,
+    })
+    if (!firstWallet?.account?.address) return
+    return firstWallet?.account?.address
```

Comment:
formatEthereumAddress

Reply:
This method is used to get the address as `mnemonicId`, sometimes you need to use isSameAddress to determine if it's the same address, so you don't need format.